Generate/build.xml

```diff
@@ -19,12 +19,8 @@
     <!-- Construct the path to the test scripts, either the locally defined dir or a new dir under "lib" -->
     <property name="testscripttools.repo.dir" location="${lib.dir}/Nictiz-tooling-testscripts/${testscripttools.version}"/>
     <condition property="testscripttools.dir" value="${testscripttools.localdir.abs}" else="${testscripttools.repo.dir}">
-            <length string="${testscripttools.localdir}" trim="true" when="greater" length="0"/>
-        </condition>
-    <!-- Just for the case when we run the build file directly from Github (see below), we have to set the location 
-         where this file expects the testscripttools repo dir to be. It isn't there yet, but it will be downloaded
-         there by the time we need it. -->
-    <property name="included.basedir" value="${testscripttools.repo.dir}" />
+        <length string="${testscripttools.localdir}" trim="true" when="greater" length="0"/>
+    </condition>
```

Comment:
Ja, dit is eigenlijk een no-brainer. Ik denk wel dat het nog ergens nuttig is om die opmerking er nog in te houden.